lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -897,7 +897,8 @@ LogicalResult FIRRTLModuleLowering::lowerPorts(
     hw::PortInfo hwPort;
     hwPort.name = firrtlPort.name;
     hwPort.type = lowerType(firrtlPort.type);
-    hwPort.sym = firrtlPort.sym;
+    hwPort.sym = firrtlPort.sym ? firrtlPort.sym.getSymName()
+                                : StringAttr::get(moduleOp->getContext(), "");
```

Comment:
Can you add a TODO comment or open an issue that field sensitive symbols are ignored at LowerToHW?

Reply:
Will open the issue. Maybe we have to extend the field sensitive symbols for HW dialect also

Reply:
Added an error here. I don't think it would be legal to ignore the field sensitive symbols here.